spec.md

```diff
@@ -558,6 +563,18 @@ message VolumeMetadata {
   // each Plugin keeps this information as small as possible.
   map<string, string> values = 1;
 }
+
+// A standard way to encode credential data. The total bytes of the values in
+// the Data field must be less than MaxSecretSize bytes.
```

Comment:
Wanted to define a const, but protobuf doesn't let you do that. So changing to a comment.